packages/fleet/generated-changes/dependencies/gitjob/dependency.yaml

```diff
@@ -1,3 +1,3 @@
 workingDir: ""
-url: https://github.com/rancher/fleet/releases/download/v0.3.6/fleet-0.3.6.tgz
```

Comment:
:eyes:

Reply:
Yep, we missed that for the first RC lol